app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,92 @@
+<p>Themes you might cover for 11 to 14 year olds (key stage 3): </p>
+<p>When teaching the lower key stages, you could use interactive exercises during your lessons, like creating a robot car.   </p>
+<ul class="vertical-tags">
+<li class="tag">Algorithms</li>
+<li class="tag">Programming and debugging   </li>
+<li class="tag">Creative projects </li>
+<li class="tag">Use of logical reasoning</li>
+<li class="tag">Computer systems</li>
+<li class="tag">Uses of technology  </li>
+<li class="tag">Online safety </li>
+</ul>
+<p>Themes you might cover for 14 to 16 year olds (key stage 4): </p>
+<p>You'll teach computer science as a GCSE qualification. You'll teach pupils how to develop their capability, creativity and knowledge in computing. </p>
+<ul class="vertical-tags">
+<li class="tag">Computational thinking</li>
+<li class="tag">Digital media and information technology  </li>
+<li class="tag">Computer networks </li>
+<li class="tag">Cyber security  </li>
+<li class="tag">Data representation </li>
+<li class="tag">Digital technology   </li>
+</ul>
+<p>For 16 to 18 year olds (key stage 5), you'll teach computer science as a A Level qualification and encourage pupils to develop a broad range of knowledge and skills, preparing them for further learning or employment, with themes like:  </p>
+<ul class="vertical-tags">
+<li class="tag">Abstraction   </li>
+<li class="tag">Writing programmes to solve problems </li>
+<li class="tag">Efficiency of algorithms </li>
+<li class="tag">Data structures and the use of databases</li>
+<li class="tag">Systems architecture </li>
+<li class="tag">Computing related mathematics</li>
+</ul>
+<p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">Read the full national curriculum for more information about teaching computing in secondary schools</a>.</p>
+</div>
+</section>
+<section class="col col-space-m col-720">
+<h2 class="heading-s">Funding for training to teach computing </h2>
+<p>To train to be a computing teacher, you'll need to do a teacher training course. </p>
+<p>Tax-free bursaries of £28,000 and scholarships of £30,000 are available for <a href="/funding-and-support/scholarships-and-bursaries">eligible trainee computing teachers</a>. </p>
+<p>You can receive this alongside a tuition fee and maintenance loan. </p>
+<p>You may also be able to get extra funding support if you are: </p>
+<ul>
+<li>a parent or carer </li>
+<li>disabled </li>
+</ul>
+<p>Non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible for funding. </p>
+<h2 class="heading-s">Check your qualifications</h2>
+<p>You'll need a bachelors degree before training to teach, but it does not have to be in computing.</p>
+<p> Besides computing, the most popular degrees people have before doing a teacher training course in computing are:</p>
```

Comment:
Do we just want to point out the most popular degrees? On the physics page, it is more about `varied degrees` - is there a danger that we could put some people off by phrasing it this way?

Reply:
@gemmadallmandfe See what you mean but i also don't want to mislead people, when you look at the spreadsheet of successful candidates for computing all the subjects are computing based, any others only have 1 successful candidate. Maybe if we were able to talk about interests instead, 'if you like playing around this coding...' or something but again, that's quite open ended and hard for us to say/not sure how we'd get that info!